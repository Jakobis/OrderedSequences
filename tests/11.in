22
a 1
a 19
a 20
a 194
a 291
a 61
a 15
a 1931
a 1921
a 1241
a 194
d 2
re 194
re 1921
re 20
re 15
a 191
a 102
a 45
r 102
s 3
sum