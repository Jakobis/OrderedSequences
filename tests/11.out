3
102
4058
